tests/queries/0_stateless/02932_kill_query_sleep.sh

```diff
@@ -8,6 +8,7 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 function wait_query_started()
 {
     local query_id="$1"
+    $CLICKHOUSE_CLIENT --query "SYSTEM FLUSH LOGS"
```

Comment:
Why was it flaky?

Reply:
A line below it uses `system.query_log` which can be not available yet.

Reply:
https://github.com/ClickHouse/ClickHouse/pull/55271#issuecomment-1854820934

https://s3.amazonaws.com/clickhouse-test-reports/56563/8edabdfd61a7c7c37f57379fc2a40a3d25e4e7bc/fast_test.html

Reply:
So we don't create `system.query_log` on SELECT, only on flushing? This is strange...